### PR TITLE
Fix SMSG_DISPLAY_PLAYER_CHOICE for 9.x

### PR DESCRIPTION
**Pull request by Traesh**

### CODE REVIEW

File: WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs [hunk 1]
```diff
@@ -566,5 +566,119 @@ public static void HandleQuestGiverCloseQuest(Packet packet)
...
+                    Locale = ClientLocale.PacketLocaleString,
+                    Question = question
+                }, packet.TimeSpan);
+            }
+        }
+
+        public static void ReadPlayerChoiceResponse(Packet packet, int choiceId, uint index, params object[] indexes)
+        {
+            var responseId = packet.ReadInt32("ResponseID", indexes);
+            var responseIdentifier = packet.ReadInt16("ResponseIdentifier", indexes);
```
Comment (mdX7) [inline WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs:613]: having ResponseID twice sounds kinda wrong, what is this field about?

cannot check stuff myself rn, will do at a later point

File: WowPacketParser/Store/Objects/PlayerChoice.cs [hunk 2]
```diff
@@ -80,6 +83,9 @@ public sealed class PlayerChoiceResponseTemplate : IDataModel
         [DBFieldName("RewardQuestID", TargetedDatabase.BattleForAzeroth)]
         public uint? RewardQuestID;
 
+        [DBFieldName("RewardQuestID", TargetedDatabase.Shadowlands)]
```
[the remark this answers is not in the document]
Comment (Traesh) [reply WowPacketParser/Store/Objects/PlayerChoice.cs:86]: Indeed it is

File: WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs (hunk 1 above)
Comment (Traesh) [reply WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs:613]: No idea, but it has been updated by Shauren on TrinityCore :

https://github.com/TrinityCore/TrinityCore/blob/master/src/server/game/Server/Packets/QuestPackets.h#L688

I can also confirm this by my own reverse on IDA & by packet parsing : https://i.imgur.com/oTZYJ1Q.png

Comment (mdX7) [reply WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs:613]: didnt know we already got that in core, we may just leave it like that then

**Pull request merged**